rust/src/crypto/sign.rs

```diff
@@ -124,11 +146,11 @@ impl ByteObject for Signature {
 }
 
 impl Signature {
-    /// Length in bytes of a [`Signature`]
+    /// Length in bytes of this signature.
     pub const LENGTH: usize = sign::SIGNATUREBYTES;
 
-    /// Compute the floating point representation of the hashed
-    /// signature and ensure that it is below the given threshold:
+    /// Computes the floating point representation of the hashed signature and ensure that it is
```

Comment:
```suggestion
    /// Computes the floating point representation of the hashed signature and ensures that it is
```

Reply:
i'll fix this typo in one of the other doc rebases i'm currently on so it won't get lost.